helm-chart/splunk-kubernetes-metrics/templates/clusterRoleBinding.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: "system:heapster"
+  name: "system:nodes"
```

Comment:
I think using https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/rbac/kubelet-api-auth/kubelet-api-admin-role.yaml might be better.